### PR TITLE
Pp 40

### PR DESCRIPTION
**Pull request by retorres87**

### CODE REVIEW

File: plenigo_plugin/PlenigoShortcodeManager.php [hunk 1]
```diff
@@ -175,13 +176,44 @@ public function plenigo_handle_shortcode($atts, $content = null, $tag = null) {
...
+     *
+     * @param $atts attributes
+     * @param null $content content
+     * @param null $tag shortcode tag
+     *
+     * @return string rendered snippet
+     */
+    public function plenigo_handle_login_shortcode($atts, $content = null, $tag = null) {
+        $a = shortcode_atts(array(
+            'title' => "Abmelden",
```
Comment (robertholfeld) [inline plenigo_plugin/PlenigoShortcodeManager.php]: Abmelden?

File: plenigo_plugin/PlenigoShortcodeManager.php [hunk 2]
```diff
@@ -175,13 +176,44 @@ public function plenigo_handle_shortcode($atts, $content = null, $tag = null) {
...
+        $loginSnippet = PlenigoSDKManager::get()->getLoginSnippet();
+
+        if (empty($targetUrl) && isset($_GET["redirect"])) {
+            $targetUrl = $_GET["redirect"];
+        }
+
+        if (!empty($targetUrl)) {
+            $_SESSION["plenigo_login_redirect_url"] = $targetUrl;
+        }
+        return "<a href=\"#\" onclick=\"$loginSnippet return false;\">Abmelden</a>";
```
Comment (robertholfeld) [inline plenigo_plugin/PlenigoShortcodeManager.php]: Abmelden?

File: plenigo_plugin/PlenigoShortcodeManager.php (hunk 1 above)
Comment (retorres87) [reply plenigo_plugin/PlenigoShortcodeManager.php]: Fixed so by default is an english title

File: plenigo_plugin/PlenigoShortcodeManager.php (hunk 2 above)
Comment (retorres87) [reply plenigo_plugin/PlenigoShortcodeManager.php]: Fixed so that it uses the title feature

**Pull request merged**